### ISSUE TITLE
Finding located on the chest

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10124]
**Preferred term label:**
Finding located on the chest

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Mondor disease

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000051
- label: Located on the chest
- parents: [Spatial pattern (HP:0012836)](https://hpo.jax.org/app/browse/term/HP:0012836)
- pmid: 
- def: Applies to a finding that is situated or occurring around the thorax (chest).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10124
- synonyms:
